pkg/protocol/engine/accounts/accounts.go

```diff
@@ -44,8 +44,7 @@ func NewAccountData(id iotago.AccountID, opts ...options.Option[AccountData]) *A
 
 func (a *AccountData) AddBlockIssuerKeys(blockIssuerKeys ...iotago.BlockIssuerKey) {
 	for _, blockIssuerKey := range blockIssuerKeys {
-		k := blockIssuerKey
-		a.BlockIssuerKeys.Add(k)
+		a.BlockIssuerKeys.Add(blockIssuerKey)
```

Comment:
Isn't the pointer we add going to change in the next iteration?

Reply:
they changed that with go 1.22

Reply:
https://go.dev/blog/loopvar-preview